add abkhazia country

SimpleCoderr opened the pull request.

ArtyomVancyan commented:
@SimpleCoderr, what about the area codes starting with 840? Aren't they Abkhazian too?

![Screen Shot 2025-06-26 at 20 58 02](https://github.com/user-attachments/assets/1c1d7d5e-6c1f-4e18-a23d-ce2b90721c20)

SimpleCoderr commented:
@ArtyomVancyan I added codes starting with 840, and also added 840 23, since some sources mention it.
![image](https://github.com/user-attachments/assets/500995d5-75ba-44d5-be9c-66cfa9355a4b)

ArtyomVancyan commented:
Before I merge, we have another problem. Abkhazia is still considered a part of Georgia. Although there are Russian phone numbers in Abkhazia, the flag of the country is missing from the global countries' list. See [stylesheet.json](https://github.com/typesnippet/react-phone-hooks/blob/master/resources/stylesheet.json). If you can find the flag and add the styles with the right coordinates, we can merge this. Otherwise, this will be redundant, as when typing an Abkhazian phone number, it won't show a flag.

ArtyomVancyan commented:
Yeah, it will be inappropriate. Currently, if you type the Georgian phone numbers for Abkhazia, it will show the Georgian flag, and it will work similarly for Russian phone numbers.

Thanks anyway for the suggestion and your last contribution :)
